logging/logging.go

```diff
@@ -36,6 +36,14 @@ type Logger interface {
 	IsVerbose() bool
 }
 
+// ConfigurableLogger defines behavior required by the Pack cobra.Command
+type ConfigurableLogger interface {
```

Comment:
TL;DR: Move to `cmd` package.

This interface makes sense but going with `go` idioms this should be defined in the package it is used. This specially makes sense to move given the fact that there is a comment about where it's being used.

Reply:
Done.